mixnode/src/commands/describe.rs

```diff
@@ -0,0 +1,55 @@
+use crate::config::Config;
+use crate::node::node_description::NodeDescription;
+use clap::{App, Arg, ArgMatches};
+use colored::Colorize;
+use config::NymConfig;
+use std::io;
+use std::io::Write;
+
+pub fn command_args<'a, 'b>() -> App<'a, 'b> {
+    App::new("describe")
+        .about("Describe your mixnode and tell people why they should delegate stake to you")
+        .arg(
+            Arg::with_name("id")
+                .long("id")
+                .help("The id of the mixnode you want to describe")
+                .takes_value(true)
+                .required(true),
+        )
+}
+
+pub fn execute(matches: &ArgMatches) {
```

Comment:
I think it should also first try to load the configuration file to make sure the node actually exists before attempting to describe it.
As a side node, perhaps the functionality of `describe` should also be available during `init`? Maybe as flags or something?

Reply:
I feel like there's already enough going on during `init`, adding three more fields to that, or somehow grabbing keyboard input during that process, starts to feel a bit heavyweight to me. But I'm really not sure - let's see how people deal with `describe` and then make some further decisions once we can see some real usage. 

I have added a check for the directory at start now